llvm/lib/Target/DirectX/DXIL.td

```diff
@@ -683,6 +685,14 @@ def Dot4 :  DXILOp<56, dot4> {
   let attributes = [Attributes<DXIL1_0, [ReadNone]>];
 }
 
+def CreateHandle : DXILOp<57, createHandle> {
+  let Doc = "creates the handle to a resource";
+  // ResourceClass, RangeID, Index, NonUniform
+  let arguments = [Int8Ty, Int32Ty, Int32Ty, Int1Ty];
+  let result = HandleTy;
+  let stages = [Stages<DXIL1_0, [all_stages]>];
```

Comment:
This should be invalid starting in DXIL_1_6 I think, right? Did we add a way to express that in the TD definition?

Reply:
Yeah that sounds right. I've added `Stages<DXIL1_6, [removed]>` to indicate this, but there doesn't really seem to be a good way to test this - since we have logic that checks the dxil version before deciding what to lower to, it's just impossible to generate the wrong one.